libparsec/crates/platform_device_loader/tests/units/utils.rs

```diff
@@ -6,59 +6,76 @@ pub use platform::*;
 mod platform {
     use std::path::Path;
 
-    pub fn create_device_file(path: &Path, content: &[u8]) {
-        std::fs::create_dir_all(path.parent().unwrap()).unwrap();
-        std::fs::write(path, content).unwrap();
+    pub async fn create_device_file(path: &Path, content: &[u8]) {
+        tokio::fs::create_dir_all(path.parent().unwrap())
```

Comment:
Since this is a test helper, it is okay to use the sync api to access the filesystem here l. I guess you changed to the Tokio one given the function was required to be asynchronous anyway due to constraints on the web implémentation ?
(If that's the case I'm okay with it ^^)

Reply:
Yes it's now async given the constrain of the web implementation.